src/app/debug/debug-tree/debug-tree.component.ts

```diff
@@ -48,7 +49,7 @@ export class DebugTreeComponent implements OnDestroy {
   showMultipleAtATime!: boolean;
   showMultipleAtATimeSubscription!: Subscription;
 
-  private _currentView: any;
+  private _currentView?: CurrentView;
```

Comment:
This should not be possibly undefined, also why is this a new type: `CurrentView` instead of [`View`](https://github.com/wearefrank/ladybug-frontend/blob/master/src/app/shared/interfaces/view.ts)?